.github/scripts/release.sh

```diff
@@ -8,19 +8,25 @@ err () {
 }
 
 usage () {
-    err "usage: release.sh [-h] [-V <version>] <pkgdir> [<pkgdir>...]"
+    err "usage: release.sh [-h] [-V <version>] [-p] [-m <message>] <pkgdir> [<pkgdir>...]"
     err
     err "Prepare a release by updating files in this repo."
-    err "Run this prior to publishing to NPM."
+    err "Run this prior to publishing to NPM (or elsewhere, e.g. DevTools)."
     err ""
     err "    -V   the new NPM version"
+    err "    -p   push the bump commit (no need if publish.sh is called afterwards)"
+    err "    -m   improve the commit message by specifying what was bumped"
     err ""
 }
 
 VERSION=
-while getopts V:h flag; do
+PUSH_COMMIT="false"
+COMMIT_MESSAGE="Bump to "
+while getopts V:p:h flag; do
```

Comment:
You'll have to add `m:` here. `m` being the flag, and `:` meaning that the flag takes an argument, i.e. take the next command line argument and put it in `$OPTARG`.

```suggestion
while getopts V:p:m:h flag; do
```

Without this, you'll get a `unknown flag -m` like error.